dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -454,24 +454,56 @@ class StartupTest {
 
   Future<TaskResult> run() async {
     return await inDirectory<TaskResult>(testDirectory, () async {
-      final String deviceId = (await devices.workingDevice).deviceId;
-      await flutter('packages', options: <String>['get']);
-
-      const int iterations = 15;
+      final Device device = await devices.workingDevice;
+      const int iterations = 5;
```

Comment:
\cc @liyuqian re: https://github.com/flutter/flutter/pull/67147
Upping from 3 and then 15 significantly increased the amount of time it took this test to run.

We also suspect it's the cause for the temp storage filling up on Android (hence the uninstall at the end of every loop).